hack/generate-proto.sh

```diff
@@ -24,8 +24,9 @@ docker run --rm gen-proto cat v1/skaffold.pb.go > proto/v1/skaffold.pb.go
 docker run --rm gen-proto cat v1/skaffold.pb.gw.go > proto/v1/skaffold.pb.gw.go
 
 # Copy v2 files
-docker run --rm gen-proto cat v2/skaffold.pb.go > proto/v2/skaffold.pb.go
-docker run --rm gen-proto cat v2/skaffold.pb.gw.go > proto/v2/skaffold.pb.gw.go
+docker run --rm gen-proto ls /proto/
```

Comment:
need this?

Reply:
oh shoot, forgot to remove that 😅 thanks for catching it